functests/performance/updating_profile.go

```diff
@@ -133,6 +133,29 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernerl is default when realTimeKernel field does not exist", func() {
+			timeout := time.Duration(len(workerRTNodes) * 10)
+
+			profile.Spec.RealTimeKernel = nil
+
+			By("Applying changes in performance profile")
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+
+			if profile.Spec.RealTimeKernel == nil {
+				sum := 0
+				for i := 1; i < 5; i++ {
```

Comment:
I think Gomega's `https://onsi.github.io/gomega/#consistently` is what you might want to use here

Reply:
Sure, I will check this out. Thanks.